tests/lean/run/array1.lean

```diff
@@ -14,7 +14,7 @@ a.foldl Nat.add 0
 #exit
```

Comment:
I didn't remove this `#exit`; `Array.mk` is not a primitive anymore and makes the interpreter segfault.

Reply:
Thanks for letting me know. I will fix this.